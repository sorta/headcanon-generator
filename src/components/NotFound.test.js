import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import NotFound from './NotFound';

// Tests for component
describe("NotFound with no subject or descriptor", () => {

  it('renders and matches our snapshot', () => {
    const snapshot = renderer.create(
      <NotFound />
    );
    const tree = snapshot.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('has an h2 element', () => {
    const component = shallow(<NotFound />);
    expect(component.find('h2').exists()).toBe(true);
    expect(component.find('h2').text()).toBe('Not Found!');
  });
});
