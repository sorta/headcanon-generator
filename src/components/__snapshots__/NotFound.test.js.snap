// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound with no subject or descriptor renders and matches our snapshot 1`] = `
<div>
  <h2>
    Not Found!
  </h2>
</div>
`;
